src/main/resources/application.yml

```
@@ -21,6 +21,7 @@ spring:
             token-uri: \${AUTH_CLIENT_TOKEN_URI:}
             user-name-attribute: sub
             jwk-set-uri: \${AUTH_CLIENT_JWK_URI:}
+white-list-users: \${WHITE_LIST_USERS:}
```

use semantic names that help to recognize also the content!
```suggestion
white-list-cf-users: \${WHITE_LIST_CF_USERS:}
```